app/components/works/contributor_row_component.html.erb

```diff
@@ -1,16 +1,16 @@
-<div class="row inner-container" data-controller="contributors" data-contributors-required-value="<%= author? %>">
+<div class="row inner-container contributor-row" data-controller="contributors" data-contributors-required-value="<%= author? %>">
   <div class="col-md-11">
     <div class="row">
       <div class="col-md-3">
         <%= form.label :role_term, role_term_label, class: 'form-label' %>
         <%= render PopoverComponent.new key: 'work.role_term' %>
         <%= select_role %>
       </div>
-      <div class="col-md-1 role-prefix">
+      <div class="col-sm-1 role-prefix">
```

Comment:
This switch causes this to get awkwardly positioned:

<img width="569" alt="Screen Shot 2021-08-23 at 12 17 07 PM" src="https://user-images.githubusercontent.com/92044/130489442-9b41cfde-ec93-42a7-90a2-59cf6620821e.png">

It looked like this before.
<img width="554" alt="Screen Shot 2021-08-23 at 12 15 06 PM" src="https://user-images.githubusercontent.com/92044/130489454-6cafa26b-b167-47b2-814b-f1994e86baa4.png">

Reply:
Switched back to `col-md-1`.